### ISSUE TITLE
Environmental Sway With Head Turn

### ISSUE THREAD

**Issue by sheepamonggoats** [Nibre/MotherVR#11]
Not sure if you plan to address this it seems the environment especially objects close will draw in or out with up down head movement or sway left and right with the same head turns. It's as if the environment is attached to your fact by a piece of string.

**Comment by MulleredOG** [comment]
+1 for this.  The issue seems linked to the character / avatar movement. If you look down at your feet on the floor when turning you can see that the character turns on a circle rather than pivoting from a point. (you need to have a standard FOV set to see your feet)  It looks like this circular turn movement is translated to the head.  Not sure how easy this would be to sort but I hope it can be fixed as I find it totally immersion breaking

**Comment by mamefan** [comment]
Agreed. When I turn my head, the world moves with it, and it makes this pretty unplayable for me.
